src/util/misc.rs

```diff
@@ -43,6 +44,7 @@ mod message_signing {
     use util::address::{Address, AddressType};
 
     /// An error used for dealing with Bitcoin Signed Messages.
+    #[cfg_attr(docsrs, doc(cfg(feature = "secp-recovery")))]
```

Comment:
TBH, I somewhat blindly added the attribute to each instance of `#[cfg]` I `ripgrep`ed. I could remove it but I'm also thinking keeping it may be good way to avoid accidentally messing up in refactors.